src/docs/development/platform-integration/platform-channels.md

```diff
@@ -646,3 +661,66 @@ classes, or create your own codec.
 [BinaryCodec]: {{site.api}}/flutter/services/BinaryCodec-class.html
 [StringCodec]: {{site.api}}/flutter/services/StringCodec-class.html
 [JSONMessageCodec]: {{site.api}}/flutter/services/JSONMessageCodec-class.html
+
+## Channels and Platform Threading
+
+When writing code on the platform side, which is the Android/iOS side using
```

Comment:
subjective/optional style nit: This could be rewritten to use [active voice](https://developers.google.com/style/voice). Also, leading with prepositional phrases like this makes the structure of the sentence kind of hard to follow since the subject is lurking at the end.

I suggest an alternative like:

> Invoke all channel methods on the platform's main thread when writing code on the platform side.

(Not sure what the value is of specifying that "platform" means "Android/iOS.")

This applies to a couple other sentences below.

Reply:
Very good catch, @mklim.